Examples/Algorithms/TrackFitting/src/KalmanFitterFunction.cpp

```diff
@@ -129,14 +130,18 @@ ActsExamples::makeKalmanFitterFunction(
   cfg.resolvePassive = false;
   cfg.resolveMaterial = true;
   cfg.resolveSensitive = true;
-  Acts::Navigator navigator(cfg);
-  Propagator propagator(stepper, std::move(navigator));
-  Fitter trackFitter(std::move(propagator));
+  Acts::Navigator navigator(cfg, logger.cloneWithSuffix("Navigator"));
+  Propagator propagator(stepper, std::move(navigator),
+                        logger.cloneWithSuffix("Propagator"));
+  Fitter trackFitter(std::move(propagator), logger.cloneWithSuffix("Fitter"));
```

Comment:
Maybe we could name the Fitter "KF" instead of "Fitter"? Same with "DirectKF" below. Might be clearer, and also is shorter.

Reply:
my idea here was that the parent logger already has the "Kalman" prefix so it will propagate through to the child loggers

Reply:
https://github.com/acts-project/acts/pull/1885/files#diff-a0346d9527c4778d50b91394c01430aad0f56675c5f533ca09503f85c7c827a1R77

Reply:
Ah I see